src/pages/Lending/Pool/components/Borrow/BorrowConfirm.tsx

```diff
@@ -128,6 +139,42 @@ export const BorrowConfirm = ({
     return TxStatus.Unknown
   }, [loanTxStatus])
 
+  const mixpanel = getMixPanel()
+  const eventData = useMemo(() => {
+    if (!confirmedQuote) return {}
+
+    const assets = selectAssets(store.getState())
+    const compositeBorrowAsset = getMaybeCompositeAssetSymbol(borrowAsset?.assetId ?? '', assets)
+    const compositeCollateralAsset = getMaybeCompositeAssetSymbol(collateralAssetId ?? '', assets)
+
+    return {
+      borrowAsset: compositeBorrowAsset,
+      collateralAsset: compositeCollateralAsset,
+      borrowAssetChain: borrowFeeAsset?.networkName,
+      collateralAssetChain: collateralFeeAsset?.networkName,
+      totalFeesUserCurrency: bn(confirmedQuote.quoteTotalFeesFiatUserCurrency).toFixed(2),
+      totalFeesUsd: bn(confirmedQuote.quoteTotalFeesFiatUsd).toFixed(2),
+      depositAmountCryptoPrecision: depositAmount,
+      collateralAmountCryptoPrecision: confirmedQuote.quoteCollateralAmountCryptoPrecision,
+      collateralAmountUserCurrency: bn(
+        confirmedQuote.quoteCollateralAmountFiatUserCurrency,
+      ).toFixed(2),
+      collateralAmountUsd: bn(confirmedQuote.quoteCollateralAmountFiatUsd).toFixed(2),
+      borrowedAmountUserCurrency: bn(confirmedQuote.quoteBorrowedAmountUserCurrency).toFixed(2),
+      borrowedAmountUsd: bn(confirmedQuote.quoteBorrowedAmountUsd).toFixed(2),
+      borrowedAmountCryptoPrecision: confirmedQuote.quoteBorrowedAmountCryptoPrecision,
+      debtAmountUserCurrency: bn(confirmedQuote.quoteDebtAmountUserCurrency).toFixed(2),
+      debtAmountUsd: bn(confirmedQuote.quoteDebtAmountUsd).toFixed(2),
+    }
+  }, [
+    borrowAsset?.assetId,
+    borrowFeeAsset?.networkName,
+    collateralAssetId,
+    collateralFeeAsset?.networkName,
+    confirmedQuote,
+    depositAmount,
+  ])
+
```

Comment:
i wonder if it'd be worth stuffing this into a hook so it can be reused in `BorrowInput`?

Reply:
see `src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useMixpanel.tsx`

Reply:
As discussed on Discord, not a fan of early abstracting this just for the sake of saving duplication in one place - additionally, hooks bring re-render complexity that we don't want in the input variant, the confirm variant is already reactive so would be realistically no change there